### PR TITLE
pythonPackages.sortedcontainers: 1.5.7 -> 2.0.4

### PR DESCRIPTION
**Pull request by costrouc**
###### Motivation for this change

Moving sortedcontianers from top-level -> python-modules and updating
to newest sortedcontianers release.

###### Things done

pythonPackages.sortedcontainers: 1.5.7 -> 2.0.4

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [X] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by costrouc** [comment]
I will need to add myself as a maintainer before merging

**Comment by costrouc** [comment]
Requires myself to be added to maintainers (pull request #44655) list before this pull request
Merge state

**Comment by xeji** [comment]
@GrahamcOfBorg eval

File: pkgs/development/python-modules/sortedcontainers/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/sortedcontainers/default.nix]: no quotes

File: pkgs/development/python-modules/sortedcontainers/default.nix
Comment (dotlambda) [inline pkgs/development/python-modules/sortedcontainers/default.nix]: That's not true.
```
checkPhase = ''
  py.test
''
```
should suffice.

**Review by dotlambda — CHANGES_REQUESTED** [review]

File: pkgs/development/python-modules/sortedcontainers/default.nix
Comment (costrouc) [reply pkgs/development/python-modules/sortedcontainers/default.nix]: Those lines were left over from the original. The Pypi tarball does not come with tests. So I have now set `doCheck = false;` with appropriate comment.

**Comment by dotlambda** [comment]
@GrahamcOfBorg build python2.pkgs.sortedcontainers python3.pkgs.sortedcontainers

**Comment by costrouc** [comment]
@GrahamcOfBorg build python2.pkgs.sortedcontainers python3.pkgs.sortedcontainers

**Comment by costrouc** [comment]
Requested changes have been made and commit and messages have been corrected.

**Pull request merged**
